### PR TITLE
Router

### PR DESCRIPTION
**Pull request by Julianhm9612**
## Summary

- Added router library
- Configure router library and add login and signup routes

## Screenshots

router

## Trello Card

https://trello.com/c/iOWe3RDC/25-routing-i-router

### CODE REVIEW

File: src/screens/Layout/index.tsx [hunk 1]
```diff
@@ -14,6 +15,14 @@ function Layout({ children }: Props) {
     <div className={styles.layout}>
       <Toaster position="top-right" />
       {/* Future Header */}
+      <nav>
+        <NavLink className={({ isActive }) => (isActive ? styles.active : '')} to="/">
+          Login
+        </NavLink>
+        <NavLink className={({ isActive }) => (isActive ? styles.active : '')} to="/sign_up">
+          Sign up
+        </NavLink>
```
Comment (ebedoyaalzate) [inline src/screens/Layout/index.tsx]: This link should be the buttons of the containers, not a navbar

File: src/screens/Login/index.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+function Login() {
+  return (
+    <Container>
+      <span>Login</span>
+    </Container>
+  );
+}
+
+export default Login;
```
Comment (ebedoyaalzate) [inline src/screens/Login/index.tsx]: add empty line

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

File: src/screens/Login/index.tsx (hunk 2 above)
Comment (Julianhm9612) [reply src/screens/Login/index.tsx]: Added :suspect:

File: src/screens/Layout/index.tsx (hunk 1 above)
Comment (Julianhm9612) [reply src/screens/Layout/index.tsx]: Removed temporary nav and added links on buttons :suspect:

File: src/screens/Login/index.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import Container from 'components/Container';
+import Button from 'components/Button';
+
+function Login() {
+  const { t, i18n } = useTranslation();
+
+  return (
+    <Container>
+      <Link to="/sign_up">
+        <Button label={t('Login:lblSignUp')} loading={false} />
```
Comment (ebedoyaalzate) [inline src/screens/Login/index.tsx]: This button is not necessary, just leave the Link component

File: src/screens/SignUp/index.tsx [hunk 4]
```diff
@@ -92,7 +93,9 @@ function SignUp() {
         />
         <Button type="submit" label={t('SignUp:lblSignUp')} loading={isLoading} primary />
       </form>
-      <Button label={t('SignUp:lblLogin')} onClick={() => reset()} loading={false} />
+      <Link to="/">
+        <Button label={t('SignUp:lblLogin')} onClick={() => reset()} loading={false} />
```
Comment (ebedoyaalzate) [inline src/screens/SignUp/index.tsx]: same

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

File: src/screens/SignUp/index.tsx (hunk 4 above)
Comment (Julianhm9612) [reply src/screens/SignUp/index.tsx]: The button component has been removed  :suspect:

File: src/screens/Login/index.tsx (hunk 3 above)
Comment (Julianhm9612) [reply src/screens/Login/index.tsx]: The button component has been removed :suspect:

**Review by ebedoyaalzate — APPROVED** [review]

**Pull request merged**